static/CSS/home.css

```
@@ -4,6 +4,11 @@
     al igual que bloqueamos que se repita y le ponemos el artributo fixed
     para que no se mueva y simule el efecto parallax.
 */
+
+@font-face{
+    font-family: 'K2D';
+    src:url('/static/fonts/K2D-Regular')
```

Me metí en https://ceicsite-dev-pr-14.herokuapp.com/ y las fuentes no se cargan correctamente. Creo que para que funcionen puedes cambiar esta línea a esto:
```suggestion
    src:url('../fonts/K2D-Regular')
```

---

@lapy0110